### PR TITLE
Changed general spanish translations

### PR DESCRIPTION
**Pull request by jerodb**
## What does this PR do?
Some general spanish translations were in english. 

## How do I test this PR?
Check changed spanish translations in locals/es.yml

### CODE REVIEW

File: locales/es.yml [hunk 1]
```diff
@@ -62,16 +63,16 @@ es:
     reactions: 'reacciones'
     story: 'Artículo'
   flagged_usernames:
-    notify_approved: '{0} approved username {1}'
-    notify_rejected: '{0} rejected username {1}'
-    notify_flagged: '{0} reported username {1}'
-    notify_changed: 'user {0} changed their username to {1}'
+    notify_approved: '{0} nombre de usuario aprobado {1}'
+    notify_rejected: '{0} nombre de usuario rechazado {1}'
+    notify_flagged: '{0} nombre de usuario reportado {1}'
```
Comment (kgardnr) [inline locales/es.yml]: These -ed words are verbs not nouns, the sentence in English with more context would be:

"Kim Gardner approved username myname124"

So I think it should be:

- notify_approved: '{0} ha aprovado nombre de usuario {1}'
- notify_rejected: '{0} ha rechazado nombre de usuario {1}'
- notify_flagged: '{0} ha reportado nombre de usuario {1}'

What do you think??

File: locales/es.yml [hunk 2]
```diff
@@ -62,16 +63,16 @@ es:
...
+    notify_changed: 'El usuario {0} cambio su nombre de usuario por {1}'
```
Comment (kgardnr) [inline locales/es.yml]: Shouldn't it be:

- ha cambiado OR cambió

Feedback welcome!

File: locales/es.yml [hunk 3]
```diff
@@ -193,10 +194,10 @@ es:
     minutes_plural: "minutos"
   email:
     suspended:
-      subject: "Your account has been suspended"
+      subject: "Su cuenta fue suspendida"
     banned:
-      subject: "Your account has been banned"
-      body: "In accordance with The Coral Project’s community guidelines, your account has been banned. You are now longer allowed to comment, flag or engage with our community."
+      subject: "Su cuenta fue bloqueada"
+      body: "De acuerdo con las guías de comunidad de The Coral Project, su cuenta a sido bloqueada. No podrá hacer comentarios, reportar o entrar en contacto con nuestra comunidad."
```
Comment (kgardnr) [inline locales/es.yml]: "ha sido bloqueada", no?

File: locales/es.yml [hunk 4]
```diff
@@ -295,24 +296,24 @@ es:
...
-    notify_edited: '{0} edited comment "{1}"'
-    notify_accepted: '{0} accepted comment "{1}"'
-    notify_rejected: '{0} rejected comment "{1}"'
-    notify_flagged: '{0} flagged comment "{1}"'
-    notify_reset: '{0} reset status of comment "{1}"'
+    notify_edited: '{0} comentarios editados "{1}"'
+    notify_accepted: '{0} comentarios aceptados "{1}"'
+    notify_rejected: '{0} comentarios rechazados "{1}"'
+    notify_flagged: '{0} comentarios reportados "{1}"'
+    notify_reset: '{0} resetear el status de comentarios "{1}"'
```
Comment (kgardnr) [inline locales/es.yml]: Same here, with context it's:

Kim Gardner edited comment "Blah blah blah"

- notify_edited: '{0} ha editado el comentario "{1}"'
- notify_accepted: '{0} ha aceptado el comentario "{1}"'
- notify_rejected: '{0} ha rechazado el comentario "{1}"'
- notify_flagged: '{0} ha reportado el comentario "{1}"'
- notify_reset: '{0} ha reseteado el status del comentario "{1}"'

Feedback on tense, etc, welcome!

**Review by kgardnr — CHANGES_REQUESTED** [review]

**Comment by kgardnr** [comment]
Thank you so much @jerodb for these contributions! I added some feedback on this one - comments/feedback welcome!

I will merge the others :) Thank you again!

**Comment by kgardnr** [comment]
Hey @jerodb any thoughts on my feedback? We would love to get this in our next release, but we would need it by Tuesday fyi

**Comment by jerodb** [comment]
Hi @kgardnr , I completely agree with the changes you propose. I've already modify the pull request with them. Please let me know if there is anything else that should be changed.

File: locales/es.yml (hunk 1 above)
Comment (jerodb) [reply locales/es.yml]: this is correct @kgardnr !

**Comment by kgardnr** [comment]
Awesome @jerodb! Thank you SO MUCH!!! I'll merge this shortly :)

File: locales/es.yml (hunk 3 above)
Comment (jerodb) [reply locales/es.yml]: yes. I also changed
```
suspended:
	subject: "Su cuenta ha sido suspendida"
```

**Review by kgardnr — APPROVED** [review]

File: locales/es.yml (hunk 4 above)
Comment (jerodb) [reply locales/es.yml]: agree

**Pull request merged**